esp32-hal/examples/rtc_watchdog.rs

```diff
@@ -0,0 +1,66 @@
+//! This demos the RTC Watchdog Timer (RWDT).
+//! The RWDT is initially configured to trigger an interrupt after a given
+//! timeout. Then, upon expiration, the RWDT is restarted and then reconfigured
+//! to reset both the main system and the RTC.
+
+#![no_std]
+#![no_main]
+
+use core::cell::RefCell;
+
+use esp32_hal::{
+    clock::ClockControl,
+    interrupt,
+    pac::{self, Peripherals},
+    prelude::*,
+    Rtc,
+};
+use esp_hal_common::Rwdt;
```

Comment:
we should avoid importing anything from `esp_hal_common` in the examples since the (end) user shouldn't even know it exists - I think that also slipped through in the previous PR (... and it happens to all of us every day)

sometimes we forget to re-export the necessary types in the chip-specific HALs and sometimes we just import it wrong

Reply:
Thanks for the explanation! I've fixed this importing issue for every target, including ESP32-C3 from the previous PR.